pulp_docker/app/tasks/sync_stages.py

```diff
@@ -77,44 +76,36 @@ def __init__(self, remote):
                 to_download.append(downloader.run(extra_data={'headers': V2_ACCEPT_HEADERS}))
                 pb.increment()
 
-        pb_parsed_tags = ProgressBar(message='Processing v2 Tags', state='running')
-        pb_parsed_ml_tags = ProgressBar(message='Parsing Manifest List Tags', state='running')
-        pb_parsed_m_tags = ProgressBar(message='Parsing Manifests Tags', state='running')
-        global pb_parsed_blobs
-        pb_parsed_blobs = ProgressBar(message='Parsing Blobs', state='running')
```

Comment:
i was thinking we would leave the counter for blobs and manifests.
that would require merging pb_parsed_ml_tags and pb_parsed_m_tags and pb_parsed_man into 1 variable. So for example you would call this counter each time after `create_tagged_manifest_list` , `create_tagged_manifest` and `create_manifest`. Wdyt?

Reply:
@ipanova and I chatted about this and decided to leave these progress bars out for now. If more information is helpful, we can add it in later.